### PR TITLE
ADD NEW SJ LOGO TO MANAGER

### PR DESCRIPTION
**Pull request by Reinkaos**
Acceptance Criteria

The text "Supplejack" in the header is replaced with a suitable variant of the "Logo – Horizontal" (see note below). I'm happy with the black banner getting only slight taller only if necessary.
Add favicon as per Sj docs (http://digitalnz.github.io/supplejack/)

### CODE REVIEW

File: app/assets/stylesheets/forms.scss [hunk 1]
```diff
@@ -64,13 +64,13 @@ fieldset#management_permissions , fieldset#harvesting_permissions {
 
 .pause, .resume {
   background:none!important;
-  color: #2ba6cb;
-  border:none; 
+  color: #118a0f;
```
Comment (motizuki) [inline app/assets/stylesheets/forms.scss]: can we use the var `$mainColor` here?

File: app/assets/stylesheets/top_nav.scss [hunk 2]
```diff
@@ -1,23 +1,38 @@
...
+    width: 11.875rem;
+  }
+
+  .supplejack-logo {
+    padding: 0;
+  }
 }
 
 .top-bar {
+  /*height: 60px;*/
```
Comment (motizuki) [inline app/assets/stylesheets/top_nav.scss]: remove comment

**Comment by motizuki** [comment]
👍

File: app/assets/stylesheets/forms.scss (hunk 1 above)
Comment (Reinkaos) [reply app/assets/stylesheets/forms.scss]: Yes, yes we can.

File: app/assets/stylesheets/top_nav.scss (hunk 2 above)
Comment (Reinkaos) [reply app/assets/stylesheets/top_nav.scss]: 💃

File: app/assets/stylesheets/devise.css.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+.devise-block {
+	margin-top: 30px !important;
```
Comment (DanHenton) [inline app/assets/stylesheets/devise.css.scss]: is the `!important` needed?

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -18,6 +18,14 @@ http://digitalnz.org/supplejack
     <%= javascript_include_tag "application" %>
     <%= csrf_meta_tags %>
     <%= yield(:head) %>
+
+    <%= favicon_link_tag asset_path('favicon-32x32.png'), rel: 'icon', type: 'image/png' %>
+    <%= favicon_link_tag asset_path('favicon-16x16.png'), rel: 'icon', type: 'image/png' %>
+    <%= favicon_link_tag asset_path('apple-touch-icon.png'), rel: 'apple-touch-icon', type: 'image/png', sizes: '144x144' %>
+    <%= favicon_link_tag asset_path('manifest.json'), rel: 'manifest' %>
+    <%= favicon_link_tag asset_path('safari-pinned-tab.svg'), rel: 'mask-icon', color: "#118a0f" %>
```
Comment (DanHenton) [inline app/views/layouts/application.html.erb]: is there a way around using the color hex? `color: "#118a0f"`

File: app/assets/stylesheets/devise.css.scss (hunk 3 above)
Comment (Reinkaos) [reply app/assets/stylesheets/devise.css.scss]: Not sure, I will have a lookie.

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (Reinkaos) [reply app/views/layouts/application.html.erb]: Don't think so.

**Comment by DanHenton** [comment]
👍

**Pull request merged**